mesh2hrtf/NumCalc/manage_numcalc.py

```diff
@@ -341,9 +338,9 @@ def manage_numcalc(project_path=os.getcwd(), numcalc_path=None,
 
             if os.name == 'nt':  # Windows detected
                 # create a log file for all print-outs
-                LogFileHandle = open(f"NC{step}-{step}_log.txt", "w")
+                LogFileHandle = open(f"{cwd}\\NC{step}-{step}_log.txt", "w")
```

Comment:
```suggestion
                LogFileHandle = open(os.path.join(cwd, "NC{step}-{step}_log.txt", "w")
```
would be more robust

Reply:
Yes. I cheated here because this is explicitly "Windows NT" code :) 
but you are right. I included your suggestion.